### DISCUSSION TITLE
genre

### DISCUSSION THREAD

**Discussion by mbrazil05** [Marekkon5/onetagger#429] [Q&A]
i'm trying to tag my hip hop collection, which consist of popular hip hop artist ie. tupac,  busta rhymes, nikki minag. Is there a way I can tweak autotag to ensure it gets the year more accurately? I'm getting a lot fo 2024 when it should be something like 1997, or 2003, etc. I'm talking popular songs with correct tags.

**Comment by Marekkon5** [comment]
There is an option to use oldest matched release (in advanced settings), however 1T prioritizes closest match rather than oldest, so depending on platform this might not be as possible. You can try toggling the option I mentioned, or try different set of platforms.
